src/fosslight_dependency/_package_manager.py

```
@@ -88,21 +88,137 @@ def _reset_state(self):
+        return major
+
+    def _resolve_wrapper_command(self):
         if self.platform == const.WINDOWS:
             return 'gradlew.bat' if os.path.isfile('gradlew.bat') else ''
         if os.path.isfile('gradlew'):
             return './gradlew'
         return ''
 
-    def _run_gradle_output(self, cmd):
+    def _run_command_output(self, cmd):
         return subprocess.check_output(cmd, encoding='utf-8')
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)